packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -279,7 +280,18 @@ async def upload_file(
         if not upload_link:
             raise exceptions.S3InvalidPathError(s3_object)
 
-        e_tag = await _upload_file_to_link(session, upload_link, local_file_path)
+        if await is_r_clone_available(r_clone_settings):
+            e_tag = await sync_local_to_s3(

```

Comment:
Q: Why is the upload_meta_data/delete_meta_data only used by ```sync_local_to_s3``` ? This would need to be used by both use cases here (with or without RClone). Is there any reason not to do it?